modules/mesaides/files/update.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Managed by puppet from automation_script.sh 
+
+set -e
+
+MES_AIDES_ROOT=/opt/mes-aides/ops
+
+export PATH=/opt/puppetlabs/bin:$PATH
+
+echo "Your SSH connection ($1) triggered a shell script ($0)."
+case "$1" in
+    provision)
+        puppet apply $MES_AIDES_ROOT/manifests/ops.pp --verbose --modulepath=$MES_AIDES_ROOT/modules
+        puppet apply $MES_AIDES_ROOT/manifests/default.pp --verbose --modulepath=$MES_AIDES_ROOT/modules
+        ;;
+    deploy)
+        puppet apply $MES_AIDES_ROOT/manifests/default.pp --verbose --modulepath=$MES_AIDES_ROOT/modules
```

Comment:
Dumb*ish*, I like the fact that the logic for scripts triggered remotely is in one single place. It is still less than 25 lines. I keep as is for now and I rely on the review review to limit the risk of mistakes :-)